packages/gatsby-plugin-firebase/src/index.js

```diff
@@ -32,6 +35,7 @@ function Index({ features, children }) {
           storageBucket: process.env.GATSBY_FIREBASE_STORAGE_BUCKET,
           messagingSenderId: process.env.GATSBY_FIREBASE_MESSAGING_SENDER_ID,
           appId: process.env.GATSBY_FIREBASE_APP_ID,
+          measurementId: process.env.GATSBY_FIREBASE_MEASUREMENT_ID ? process.env.GATSBY_FIREBASE_MEASUREMENT_ID : null,
```

Comment:
You can just use `process.env.GATSBY_FIREBASE_MEASUREMENT_ID`. I think `undefined` value is fine too when initializing the Firebase app.